File: atcoder.jp/contests/caddi2018b/tasks/caddi2018b_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const [N, H, W] = input().split(" ").map(Number);
const AB = [...Array(N)].map(
  () => input().split(" ").map(Number) as [number, number]
);

const ans = AB.filter(([a, b]) => H <= a && W <= b).length;
console.log(ans);

File: atcoder.jp/contests/caddi2018b/tasks/caddi2018b_b/main.py
from typing import List, Tuple

N, H, W = map(int, input().split())
AB: List[Tuple[int, int]] = [tuple(map(int, input().split())) for _ in range(N)]

ans = sum(H <= a and W <= b for a, b in AB)
print(ans)
